NanaZip.UI.Classic/SevenZip/CPP/7zip/UI/FileManager/ProgressDialog2.cpp

```diff
@@ -16,6 +16,13 @@
 #include "ProgressDialog2.h"
 #include "ProgressDialog2Res.h"
 
+#include <winrt/Windows.UI.Xaml.Hosting.h>
```

Comment:
Modification Mark.

And unnecessary "#include <winrt/Windows.UI.Xaml.Hosting.h>" and TODO comments.